src/main/java/de/ellpeck/rockbottom/content/SmeltingRecipeLoader.java

```diff
@@ -23,7 +23,7 @@
 import java.util.HashSet;
 import java.util.Set;
 
-public class SmeltingLoader implements IContentLoader<ICompendiumRecipe> {
+public class SmeltingRecipeLoader implements IContentLoader<ICompendiumRecipe> {
```

Comment:
You may as well make this consistent with MortarRecipeLoader and make it `implements IContentLoader<SmeltingRecipe>`